v1.5.0 seemingly not working with new short file header?

Using v1.5.0 of the Checkstyle config for the SplashScreen repo actually seems to fail on the new shortened header over some regex issue. Not spotting anywhere that version is in use yet - I wonder if it worked locally somewhere, depending on versions of something, but not in my case? 1.4.0 is fine though.

```
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Cannot set property 'header' to '(/*\n * Copyright 20\d\d MovingBlocks\n *\n * Licensed under the Apache License, Version 2.0)|(// Copyright 20\d\d The Terasology F
oundation\n// SPDX-License-Identifier: Apache-2.0)'
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.tryCopyProperty(AutomaticBean.java:237)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:191)
        at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:467)
        ... 113 more
Caused by: java.lang.reflect.InvocationTargetException
        at org.apache.commons.beanutils.PropertyUtilsBean.invokeMethod(PropertyUtilsBean.java:2128)
        at org.apache.commons.beanutils.PropertyUtilsBean.setSimpleProperty(PropertyUtilsBean.java:2109)
        at org.apache.commons.beanutils.BeanUtilsBean.copyProperty(BeanUtilsBean.java:437)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.tryCopyProperty(AutomaticBean.java:227)
        ... 115 more
Caused by: java.lang.IllegalArgumentException: line 1 in header specification is not a regular expression
        at com.puppycrawl.tools.checkstyle.checks.header.RegexpHeaderCheck.postProcessHeaderLines(RegexpHeaderCheck.java:336)
        at com.puppycrawl.tools.checkstyle.checks.header.AbstractHeaderCheck.loadHeader(AbstractHeaderCheck.java:171)
        at com.puppycrawl.tools.checkstyle.checks.header.AbstractHeaderCheck.setHeader(AbstractHeaderCheck.java:149)
        at com.puppycrawl.tools.checkstyle.checks.header.RegexpHeaderCheck.setHeader(RegexpHeaderCheck.java:359)
        ... 119 more
Caused by: java.util.regex.PatternSyntaxException: Unclosed group near index 3
(/*
   ^
        at com.puppycrawl.tools.checkstyle.checks.header.RegexpHeaderCheck.postProcessHeaderLines(RegexpHeaderCheck.java:331)
        ... 122 more
```

--- a/checkstyle/checkstyle.xml
+++ b/checkstyle/checkstyle.xml
@@ -39,7 +39,7 @@
     <!-- Checks that a source file begins with a specified header. --> 
     <module name="RegexpHeader">
         <property name="severity" value="warning"/>
-        <property name="header" value="(/*\n * Copyright 20\d\d MovingBlocks\n *\n * Licensed under the Apache License, Version 2.0)|(// Copyright 20\d\d The Terasology Foundation\n// SPDX-License-Identifier: Apache-2.0)"/>
+        <property name="header" value="// Copyright 20\d\d The Terasology Foundation\n// SPDX-License-Identifier: Apache-2.0"/>
         <property name="fileExtensions" value="java"/>
     </module>
   

